<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>G4_47</class>
 <widget class="QWidget" name="G4_47">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>G4_47</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>811</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'幼圆'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt;&quot;&gt;题目：编写一个工资管理程序，将雇员类作为所有类的基类，其派生类包括经理类、销售员类、计件工类和小时工类。经理的收入为固定的月薪；销售员的收入是一小部分的基本工资加上销售额的提成；计件工的收入完全取决于其生产的工件数量；小时工的收入以小时计算，再加上加班费。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>108</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>张三</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>114</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓  名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>156</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1122334455</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>身份证：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="manager">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>79</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>经理</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sale">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>79</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>销售员</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="number">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>110</y>
     <width>79</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>计件工</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hour">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>79</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>小时工</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>831</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>工    资    条</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="b_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>身份证：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="a_2_3_4_pp">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>154</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="b_4_pp">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>154</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="a_2_3_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>哈哈哈哈:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>90</y>
     <width>21</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>831</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="jie_suan">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>153</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>结  算</string>
   </property>
  </widget>
  <widget class="QLabel" name="ti_shi">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListView" name="tiao_tiao">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>831</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="b_4_pp_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>154</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="b_5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>身份证：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>jie_suan</sender>
   <signal>clicked()</signal>
   <receiver>G4_47</receiver>
   <slot>count()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>671</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manager</sender>
   <signal>clicked()</signal>
   <receiver>G4_47</receiver>
   <slot>xx_1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sale</sender>
   <signal>clicked()</signal>
   <receiver>G4_47</receiver>
   <slot>xx_2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>number</sender>
   <signal>clicked()</signal>
   <receiver>G4_47</receiver>
   <slot>xx_3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hour</sender>
   <signal>clicked()</signal>
   <receiver>G4_47</receiver>
   <slot>xx_4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>count()</slot>
  <slot>xx_1()</slot>
  <slot>xx_2()</slot>
  <slot>xx_3()</slot>
  <slot>xx_4()</slot>
 </slots>
</ui>
